ml-algorithms/build.gradle

```diff
@@ -64,8 +69,11 @@ dependencies {
     implementation 'software.amazon.awssdk:auth'
     implementation 'software.amazon.awssdk:apache-client'
     implementation 'com.amazonaws:aws-encryption-sdk-java:2.4.1'
-    implementation 'com.jayway.jsonpath:json-path:2.8.0'
+    implementation('com.jayway.jsonpath:json-path:2.8.0') {
+        exclude group: 'org.ow2.asm', module: 'asm'
+    }
     implementation group: 'org.json', name: 'json', version: '20231013'
+    implementation group: 'org.jsoup', name: 'jsoup', version: '1.16.1'
```

Comment:
I added this to feature branch ([code commit](https://github.com/opensearch-project/ml-commons/commit/1b85cffb07977a3aedb61a41637cb1b24f6cd016#diff-68650256ee78318934f0579cf938c57e39a60b4b5452b0ec6d76830cf3100940)) for some testing purpose. We can remove it when merge to main branch.

Reply:
Removed.